packages/react/test/specs/commonTests/isConformant.tsx

```diff
@@ -56,7 +54,7 @@ export default (Component, options: Conformant = {}) => {
   const componentType = typeof Component
 
   const helperComponentNames = [
-    ...[ThemeProvider, FelaTheme, Ref, RefFindNode],
```

Comment:
- `ThemeProvider` is not needed anymore as we use `wrappingComponent`
- `FelaTheme` is not needed because we use `ThemeContext` directly #1163